templates/opensshd.conf.j2

```diff
@@ -203,13 +203,14 @@ X11UseLocalhost yes
 
 {% if ssh_server_permit_environment_vars -%}
 PermitUserEnvironment yes
```

Comment:
Since this setting accepts `yes, no or a pattern-list specifying which environment variable names to accept (for example "LANG,LC_*")`, can you make this a variable, too?
E.g.

```
{% if ssh_server_permit_environment_vars -%} 
PermitUserEnvironment {{ ssh_server_permit_environment_vars }}
{% endif %}
```

(and add it to the readme).